cmd/account/governance.go

```diff
@@ -0,0 +1,193 @@
+package account
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	governance "github.com/oasisprotocol/oasis-core/go/governance/api"
+	upgrade "github.com/oasisprotocol/oasis-core/go/upgrade/api"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+
+	"github.com/oasisprotocol/cli/cmd/common"
+	cliConfig "github.com/oasisprotocol/cli/config"
+)
+
+var (
+	govCmd = &cobra.Command{
+		Use:     "governance",
+		Short:   "Governance operations",
+		Aliases: []string{"gov"},
+	}
+
+	govSubmitProposalCmd = &cobra.Command{
```

Comment:
While you're at it, I would also migrate `network governance-proposal` to `account governance show`.

Could you also add `account governance list` to list all opened proposals and proposers?

Reply:
Added both show and list, please take a look.